ipaserver/install/server/replicainstall.py

```diff
@@ -1549,6 +1550,14 @@ def install(installer):
     # Everything installed properly, activate ipa service.
     services.knownservices.ipa.enable()
 
+    # Print a warning if CA role is only installed on one server
+    if len(ca_servers) == 1:
+        print()
```

Comment:
The warning should be printed to stderr. You can use https://docs.python.org/3/library/textwrap.html#textwrap.dedent with triple quotes to create one string and then print it to ``file=sys.stderr``.